examples/react-instantsearch-hooks/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
Seems like we don't include a prettier config in the other examples, should we start including one?

Reply:
Good remark! I removed it.